signus/freedesktop/io.github.signus_game.signus.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppStreamEdit 7.0-->
+  <id>io.github.signus_game.signus</id>
+  <name>Signus: The Artefact Wars</name>
+  <summary>turn-based strategy game</summary>
+  <developer_name></developer_name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <description>
+    <p>Signus is a Czech turn-based strategy game from 1998, developed by Valacirca and released by Zima Software. Since 2002 it is free and open source software with both code and assets.</p>
+    <p>The game features 40 units: scout vehicles, tanks, artillery, helicopters, planes, ships, and support units providing repairs, refuels and ammo. There are also nine buildings and five defense towers in the game.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://signus.paan.cz/wp-content/uploads/2013/03/mise7-1.jpg</image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="1.96.0" date="2021-05-28" type="stable">
+      <url>https://github.com/signus-game/signus/releases/tag/v1.96.0</url>
+    </release>
+  </releases>
+  <url type="homepage">https://signus.paan.cz/en/</url>
```

Comment:
signus.paan.cz is a fan website unrelated to this project. Signus currently does not have any homepage other than the Github repo.

Reply:
A website is required for @flathub. Which should I use instead?